src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1699,6 +1720,11 @@ def forward(
         # Predict object classes [batch_size, num_patches, num_queries+1]
         (pred_logits, class_embeds) = self.class_predictor(image_feats, query_embeds, query_mask)
 
+        # Predict objectness
+        objectness_logits = None
+        if self.objectness_head is not None:
+            objectness_logits = self.objectness_predictor(image_feats)
+
```

Comment:
maybe add a comment mentioning:

`objectness_logits` is used to sample top tokens during training that is not currently supported. 

https://github.com/google-research/scenic/blob/a41317783b2f33f85b94f7af0860a93c4265db58/scenic/projects/owl_vit/models.py#L322-L333